### ISSUE TITLE
Repo contains sensitive information

### ISSUE THREAD

**Issue by jpzg** [ubc-cirrus-lab/parrotfish#69]
Two files in the repository contain potentially sensitive information that we need to remove before this is open-sourced.

First, `credentials.py` has a connection string to something - it looks like an Azure Insights logs resource (it was added in December). We should remove this, then find the resource and delete it or roll the keys.

Second, `spot/prices/credentials.yml` has our Infracost API key. This is its own issue since we need to document getting one as part of setup, and possibly handle lack of a key or an incorrect key. This should also be deleted and the key should be rolled.

TL;DR

- [ ] Remove `credentials.py`
- [ ] Roll keys or delete the resource used in `credentials.py`
- [ ] Remove `spot/prices/credentials.yml`
- [ ] Roll the Infracost key
